src/Language/PureScript/ModuleDependencies.hs

```diff
@@ -50,8 +50,7 @@ sortModules toSig ms = do
     return (fst <$> ms'', moduleGraph)
   where
     toGraphNode :: S.Set ModuleName -> (a, ModuleSignature) -> m ((a, ModuleSignature), ModuleName, [ModuleName])
-    toGraphNode mns m@(_, ModuleSignature _ mn ds) = do
-      let deps = ordNub (mapMaybe usedModules ds)
```

Comment:
What's the reason for removing the ordNub? Was it unnecessary?

Reply:
Oops, no, that was an oversight. I'll put it back.

Reply:
I mean, I'm genuinely unsure what it's doing, so I was hoping you know 😆.

Reply:
Actually, maybe it would make sense to use a `Set` for the `sigImports` field?

Reply:
I think it is actually necessary if we want the module graph to not contain duplicate entries (which indeed seems like a sensible thing to want).

Reply:
Does the specific order matter though. `Set` wouldn't preserve that.

Reply:
That's true. I'm not sure. Perhaps best to put the `ordNub` back to leave it as it was then.